src/NumericalAlgorithms/Interpolation/InterpolationTargetApparentHorizon.hpp

```diff
@@ -141,19 +141,30 @@ template <typename InterpolationTargetTag, typename Frame>
 struct ApparentHorizon {
   using const_global_cache_tags =
       tmpl::list<Tags::ApparentHorizon<InterpolationTargetTag, Frame>>;
-  using initialization_tags =
-      tmpl::append<StrahlkorperTags::items_tags<Frame>,
-                   tmpl::list<::ah::Tags::FastFlow,
-                              logging::Tags::Verbosity<InterpolationTargetTag>,
-                              ::ah::Tags::PreviousStrahlkorper<Frame>>,
-                   StrahlkorperTags::compute_items_tags<Frame>>;
+  using initialization_tags = tmpl::append<
```

Comment:
This type alias should not exist.  I could not understand how this could possibly work since `initialization_tags` should only be simple tags.  It turns out that as far as I can tell it is only used to fill the unused type alias `return_tag_list` and in some detail metafunctions that can now be eliminated in favor of using CreateGetTypeAliasOrDefault and if-constexpr  in `InitializeInterpolationTarget.hpp`, and (It is also present and should be removed form the KerrHorizon target.).